src/test/isolation2/sql/resgroup/resgroup_cancel_terminate_concurrency.sql

```diff
@@ -115,4 +115,28 @@ SELECT * FROM rg_concurrency_view;
 DROP ROLE role_concurrency_test;
 DROP RESOURCE GROUP rg_concurrency_test;
 
+-- test5: terminate a query waiting for a slot, that opens a local transaction on exit callback
+DROP ROLE IF EXISTS role_concurrency_test;
+-- start_ignore
+DROP RESOURCE GROUP rg_concurrency_test;
+-- end_ignore
+
+CREATE RESOURCE GROUP rg_concurrency_test WITH (concurrency=1, cpu_rate_limit=20, memory_limit=20);
+CREATE ROLE role_concurrency_test RESOURCE GROUP rg_concurrency_test;
+1:SET ROLE role_concurrency_test;
```

Comment:
@darthunix could you point to some doc about the `isolation2` tests' syntax (an example `1:` is in this line)?

Reply:
@leskin-in look at [sql_isolation_testcase.py](https://github.com/greenplum-db/gpdb/blob/300033d1c217592ff51eade7e013d76265e3efb5/src/test/isolation2/sql_isolation_testcase.py#L497).